### ISSUE TITLE
Define features of the app

### ISSUE THREAD

**Issue by olliesharp5** [douglas86/january-hackathon-team8#13]

**Comment by olliesharp5** [comment]
yes I agree, this was just a bit of a brain dump so we can for sure streamline it to fit our timeframe

**Comment by nikimacm** [comment]
Okay. I will make basic wireframes for each tab just to give us an idea of where we should go re layout etc. We can change them later when we know which aspects we are going to cut or keep.
